servers/quarkus-server/src/testFixtures/java/org/projectnessie/server/AbstractQuarkusSmoke.java

```diff
@@ -41,7 +40,6 @@
 import org.projectnessie.model.Tag;
 
 @SuppressWarnings("resource")
-@QuarkusIntegrationTest
 @ExtendWith(QuarkusNessieClientResolver.class)
```

Comment:
nit: `QuarkusNessieClientResolver` may rely on running under `QuarkusIntegrationTest`... but I do not remember that code well anymore :) so just a note, not critical.